### PR TITLE
ldap-scim-bridge chart and openldap chart for test instance

### PR DESCRIPTION
**Pull request by jmatsushita**
See SQPIT-625

A [new ldap-scim-bridge module](https://github.com/wireapp/ldap-scim-bridge) can synchronise an external LDAP directory with Wire via Scim. This PR adds helm charts to deploy the bridge and test it in QA with an openldap instance.

## Checklist

 - [x] The **PR Title** explains the impact of the change.
 - [x] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.
 - [x] If end-points have been added or changed: the **endpoint / config-flag checklist** (see Wire-employee only backend [wiki page](https://github.com/zinfra/backend-wiki/wiki/Checklists)) has been followed.
 - [x] If a schema migration has been added, I ran **`make git-add-cassandra-schema`** to update the cassandra schema documentation.
 - [ ] Section *Unreleased* of **CHANGELOG.md** contains the following bits of information:
   - [ ] A line with the title and number of the PR in one or more suitable sub-sections.
   - [ ] If /a: measures to be taken by instance operators.
   - [ ] If /a: list of cassandra migrations.
   - [ ] If public end-points have been changed or added: does nginz need upgrade?
   - [ ] If internal end-points have been added or changed: which services have to be deployed in a specific order?

### CODE REVIEW

File: charts/ldap-scim-bridge/values.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+image:
+  repository: quay.io/wire/ldap-scim-bridge
```
Comment (fisx) [inline charts/ldap-scim-bridge/values.yaml:2]: How is this being built?

docs?  Can I have something like "read documentation *here* on how to get a test instance running; then run *these* commands to start the ldap-scim-bridge tests."

**Review by fisx — CHANGES_REQUESTED** [review]

File: charts/ldap-scim-bridge/values.yaml (hunk 1 above)
Comment (jmatsushita) [reply charts/ldap-scim-bridge/values.yaml:2]: See https://github.com/wireapp/wire-server/pull/1709/files#diff-14eedc550a0f3ad7f0f359b4fa085bb2584d09d0e37b0749e8e58ade876b3f21

Comment (jmatsushita) [reply charts/ldap-scim-bridge/values.yaml:2]: > How is this being built?

Built and push manually until I have a working version.

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,7 +30,7 @@
 ## API Changes
 
 ## Features
-
+* Helm charts to deploy [ldap-scim-bridge](https://github.com/wireapp/ldap-scim-bridge) #1709
```
[the remark this answers is not in the document]
Comment (jmatsushita) [reply CHANGELOG.md:33]: Does this 11ad4f8 work ?

[the remark this answers is not in the document]
Comment (jmatsushita) [reply CHANGELOG.md]: Nevermind this c002a0df2 did it I think

File: charts/ldap-scim-bridge/values.yaml (hunk 1 above)
Comment (jmatsushita) [reply charts/ldap-scim-bridge/values.yaml:2]: Done in https://github.com/wireapp/ldap-scim-bridge/pull/8

**Review by julialongtin — APPROVED** [review]

**Pull request merged**